### ISSUE TITLE
Clue & entry encoding issue

### ISSUE THREAD

**Issue by Golpette** [AndrewRigg/Crossword#49]
I noticed that one of the clues contained "£1.05" in it, but the £ symbol is not handled correctly.

Also need to test the encoding with Spanish wordlists: can we easily use é,á,í,ñ, etc... in both clues + crossword?

**Comment by AndrewRigg** [comment]
Yes, is this an html issue?  I think there are usually codes you can use for each symbol.  Could just replace any dodgy things with the correct code.
